app/src/androidTest/java/org/mozilla/tv/firefox/ui/DOMElementFocusTest.kt

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.ui
+
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.tv.firefox.ext.toUri
+import org.mozilla.tv.firefox.helpers.AndroidAssetDispatcher
+import org.mozilla.tv.firefox.helpers.MainActivityTestRule
+import org.mozilla.tv.firefox.ui.robots.BrowserRobot
+import org.mozilla.tv.firefox.ui.robots.navigationOverlay
+
+/**
+ * Ensures that, if the EngineView loses and regains focus, the focused DOM element maintains focused.
+ * If the focused element changes, sites like YouTube TV may break. See
+ * [org.mozilla.tv.firefox.webrender.FocusedDOMElementCacheInterface] for additional details on why this is necessary.
```

Comment:
There aren't any changes to FocusedDOMElementCacheInterface, but I notice that its kdoc includes links to classes that are neither fully qualified nor imported.  Could you add those, so that future renames don't break its documentation?